cmd/di.go

```diff
@@ -842,6 +842,9 @@ func (di *Dependencies) handleHTTPClientConnections() error {
 			di.HTTPClient.Reconnect()
 			di.QualityClient.Reconnect()
 			di.BrokerConnector.ReconnectAll()
+			if err := di.EtherClient.Reconnect(); err != nil {
```

Comment:
Can we change this main method name from `handleHTTPClientConnections` to something like `handleConnStateChange`